src/clj/bluegenes_tool_store/package.clj

```diff
@@ -0,0 +1,33 @@
+(ns bluegenes-tool-store.package
+  (:require [bluegenes-tool-store.tools :as tools]
+            [cheshire.core :as cheshire]
+            [clj-http.client :as client]
+            [taoensso.timbre :refer [infof]]))
+
+(defn get-all-npm-tools
+  "Send an HTTP request to the NPMJS API to get a list of bluegenes tools."
+  []
+  (-> (client/get (str "https://api.npms.io/v2/search"
+                       "?q=scope:intermine"
+                       "+keywords:bluegenes-intermine-tool"))
+      :body
+      (cheshire/parse-string true)
+      :results))
+
+(defn install-all-npm-tools
+  "Call NPM through the shell to install all bluegenes tools from the NPM registry."
+  []
+  (let [packages (mapv (comp :name :package) (get-all-npm-tools))]
+    (tools/sync-sh-req (into ["npm" "install" "--save"] packages)
```

Comment:
given how very good npm is at breaking between versions, it might be nice to have some error handling here too :) I'd include the npm and node versions in any error messages if possible. Another possible path is that npm isn't installed or is a very old version.